extensions/xcode/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "xcode",
+  "title": "Xcode",
+  "version": "1.0.0",
+  "author": "Sven Tiigi",
+  "license": "MIT",
+  "description": "Open your recent Xcode Projects, view the latest Xcode Releases and manage your Xcode Simulators",
+  "icon": "icon.png",
+  "commands": [
+    {
+      "name": "projects.command",
+      "title": "Projects",
+      "subtitle": "Your recent Xcode Projects",
+      "description": "Search and open your recent Xcode Projects",
+      "mode": "view"
+    },
+    {
+      "name": "simulator-applications.command",
+      "title": "Apps",
```

Comment:
Maybe something like "Installed Simulator Apps" with "Xcode" as subtitle?
Alternatively you can do "Installed Apps" with "Xcode Simulators" subtitle

Reply:
Updated to "Installed Simulator Apps" as title and "Xcode" as subtitle (https://github.com/raycast/extensions/pull/14/commits/9a8e1d863499426fd85a354069115e10626de53a)